### PR TITLE
Fixes install path for appwrite.exe from windows CLI installer

### PR DESCRIPTION
**Pull request by wess**

### CODE REVIEW

File: templates/cli/install.ps1.twig [hunk 1]
```diff
@@ -20,19 +20,26 @@ $GITHUB_arm64_URL = "https://github.com/{{ sdk.gitUserName }}/{{ sdk.gitRepoName
...
 function Greeting {
     Write-Host @"
-{{ language.params.logoUnescaped | raw }}
+
+     _                            _ _           ___   __   _____ 
+    /_\  _ __  _ ____      ___ __(_) |_ ___    / __\ / /   \_   \
+   //_\\| '_ \| '_ \ \ /\ / / '__| | __/ _ \  / /   / /     / /\/
+  /  _  \ |_) | |_) \ V  V /| |  | | ||  __/ / /___/ /___/\/ /_  
+  \_/ \_/ .__/| .__/ \_/\_/ |_|  |_|\__\___| \____/\____/\____/  
+        |_|   |_|
```
Comment (TorstenDittmann) [inline templates/cli/install.ps1.twig:37]: Can we revert this back to `{{ language.params.logoUnescaped | raw }}`?

File: templates/cli/install.ps1.twig [hunk 2]
```diff
@@ -20,19 +20,26 @@ $GITHUB_arm64_URL = "https://github.com/{{ sdk.gitUserName }}/{{ sdk.gitRepoName
 ${{ spec.title | upper }}_DOWNLOAD_DIR = Join-Path -Path $env:TEMP -ChildPath "{{ language.params.executableName }}.exe"
 
 # {{ spec.title | caseUcfirst }} CLI location
-${{ spec.title | upper }}_INSTALL_DIR = Join-Path -Path $env:LOCALAPPDATA -ChildPath "{{ spec.title | caseUcfirst }}"
+${{ spec.title | upper }}_INSTALL_DIR = Join-Path -Path $env:LOCALAPPDATA -ChildPath "{{ spec.title | caseUcfirst }}/"
```
Comment (TorstenDittmann) [inline templates/cli/install.ps1.twig:23]: pretty sure for `Join-Path` we don't need a trailing slash for the `-ChildPath`

**Review by TorstenDittmann — APPROVED** [review]

**Comment by stnguyen90** [comment]
@wess is this still relevant?

**Comment by lohanidamodar** [comment]
I'm closing this, seeing another related merged PR, we can reopen if it's still relevant.

**Pull request closed without merging**